2-achiev-landing/landing-v1.css

```diff
@@ -0,0 +1,278 @@
+@media all and (max-width: 1280px)
+{
+
+	#footer
+	{
+		width: auto;
+	}
+
+	.frame-small
+	{
+		width: auto;
+	}
+
+	.frame-medium
+	{
+		width: auto;
+	}
+
+	#header
+	{
+		width: auto;
+	}
+
+	#header h1
+	{
+		margin-top: 20%;
+	}
+
+	#part2
+	{
+		width: auto;
+	}
+
+	.text p
+	{
+		margin-left: 0;
+	}
+
+	section #part1
+	{
+		width: auto;
+		padding-left: 0;
+	}
+
+	.who h2
+	{
+		margin-left: 0;
+	}
+
+	#part1
+	{
+		margin-left: 0;
+	}
+
+}
+
+section
+{
+	width: 100%;
+}
+
+
+
+body
+{
+	font-family: montserrat;
+	font-stretch: normal;
+	font-style: normal;
+	line-height: normal;
+	letter-spacing: normal;
+	background-color: #f1f1f1;
+	width: 100%;
+	margin: 0;
+}
+
+body p
+{
+	font-size: 16px;
+	font-weight: 300;
+}
+
+body h2
+{
+	font-size: 30px;
+}
+
+body h1
+{
+	font-size: 33px;
+	font-weight: 600;
+	text-align: center;
+}
+
+#header
+{
+	width: 100%;
+	height: 75px;
+}
+
+.box
+{
+	width: auto;
+	height: 75px;
+	background-color: #4a4a4a;
+}
+
+.box-img
+{
+	width: 100%;
+	height: 400px;
+}
+
+.box-img img
+{
+	background-color: rgba(74, 74, 74, 0.5);
+	width: 100%;
+	height: 399px;
+}
+
+h1 
+{
+	position: relative;
+	bottom: 268px;
+	width: 60%;
+	left: 20%;
+	color: #ffffff
+}
+.round
+{
+	position: absolute;
+	top: 0;
+	height:150px; 
+}
+.round
+{
+	width: 150px;
+	height: 150px;
+	background-color: #4a4a4a;
+	border-radius: 130px / 130px;
+	left: 45%;
+	margin-top: 0;
+}
+
+.round img
+{
+	width: 89px;
+	height: 107px;
+	margin-top: 15%;
+	margin-left: 20%;
+}
+
+ .who h2 
+{ 	
+	color: #d4a23d;
+	width: 357px;
+	margin-left: 334px;
+	margin-right: 333px;
+	margin-top: 28px;
+}
+
+#part1
+{
+	width: 924px;
+	padding-left: 15%;
+}
+
+.text p 
+{	
+	height: 120px;
+	text-align: center;
+}
+
+#part2
+{
+	width: 100%;
+	height: 433px;
+	background-color: #ffffff;
+}
+
+#part2 p
+{
+	width: 357px;
+	height: 64px;
+	color: #000000;
+	margin-left: 35%;
+}
+
+#part2 h2
+{
+	text-align: center;
+	margin-left: 334px;
+	margin-right: 333px;
+	padding-top: 59px;
+	padding-bottom: 7px;
+}
+#frames
+{
+	display: flex;
+	justify-content: space-evenly;
+}
+
+.frame
+{	
+	width: 268px;
+	height: 185px;
+	border: solid 1px #979797;
+	background-color: #d8d8d8;
+}
+
+.way-to p
+{
+	text-align: center;
+	text-transform: uppercase;
+	margin-bottom: 40px;
+}
+
+.way-to
+{
```

Comment:
Essaye de regrouper les propriétés qui vont ensemble, globalement c'est ca mais meme retour que le reste pour la façon de centrer

Reply:
( Par regrouper j'entend ne pas avoir deux blocs .way-to )